src/language-js/printer-estree.js

```diff
@@ -3511,7 +3515,12 @@ function printArgumentsList(path, options, print) {
   //    )
   // here, but not
   //    process.stdout.pipe(socket)
-  if (isFunctionCompositionFunction(node.callee) && args.length > 1) {
+  if (
+    isFunctionCompositionFunction(node.callee) &&
+    args.length > 1 &&
+    (args.some(a => a.type !== "Identifier") ||
```

Comment:
I'm not completely sure about this. This is a valid pattern:

```js
const someFn = pipe(
  firstStep,
  secondStep,
  thirdStep
);
```

You might want to call the result of the composition several times so it's a valid reason for it not to be the callee of a call expression.